dvc/config.py

```diff
@@ -109,6 +109,7 @@ class Config(object):  # pylint: disable=too-many-instance-attributes
     SECTION_CORE_ANALYTICS = "analytics"
     SECTION_CORE_CHECKSUM_JOBS = "checksum_jobs"
     SECTION_CORE_HARDLINK_LOCK = "hardlink_lock"
+    SECTION_CORE_MULTIPLE_ROOTS = "allow_multiple_roots"
```

Comment:
So this one is set in a subrepo, right? If so the naming is confusing. Maybe `subrepo = true`?

Reply:
One of the reasons why I'm asking about this particular thingy is because it might be linked to an imperfect understanding of what we are doing here in general :slightly_smiling_face: